src/style.css

```
@@ -1 +1,75 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  background-color: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo {
+  display: flex;
+  align-self: center;
+}
+
+.nav__list {
+  display: flex;
+  height: 60px;
+  align-items: center;
+  align-content: center;
+  float: left;
+  list-style: none;
+}
+
+.nav__item {
+  margin: 0 10px;
+}
```

```suggestion
.nav__item {
  margin-right: 20px;
}
```
Лучше устанавливать только с одной стороны отступы